.github/workflows/BuildPR.yml

```diff
@@ -0,0 +1,25 @@
+name: PullRequest
```

Comment:
```suggestion
name: build-mgscopebar
```

Reply:
With it, copy&paste needs some more work in the future 😉